website/src/roadmap/python-sdk.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: section
+title: "Python SDK Roadmap"
+permalink: /roadmap/python-sdk/
+section_menu: section-menu/roadmap.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Python SDK Roadmap
+
+## Python 3 Support
```

Comment:
@aaltay @tvalentyn I copied the exact text over to here. Please let me know what you think of how the flow works, and also check out the "Roadmap highlights" section and how it links to here. 
http://apache-beam-website-pull-requests.storage.googleapis.com/6718/roadmap/python-sdk/index.html

Reply:
LGTM